credential/schema/model.go

```diff
@@ -3,28 +3,104 @@ package schema
 import "fmt"
 
 const (
-	// VCJSONSchemaType https://w3c-ccg.github.io/vc-json-schemas/v2/index.html#credential_schema_definition_metadata
-	VCJSONSchemaType string = "https://w3c-ccg.github.io/vc-json-schemas/schema/2.0/schema.json"
+	// CredentialSchema2023Type https://www.w3.org/TR/vc-json-schema/#credentialschema2023
+	CredentialSchema2023Type VCJSONSchemaType = "CredentialSchema2023"
+	// JSONSchema2023Type https://www.w3.org/TR/vc-json-schema/#jsonschema2023
+	JSONSchema2023Type VCJSONSchemaType = "JsonSchema2023"
+
+	Draft202012 JSONSchemaVersion = "https://json-schema.org/draft/2020-12/schema"
+	Draft201909 JSONSchemaVersion = "https://json-schema.org/draft/2019-09/schema"
+	Draft7      JSONSchemaVersion = "https://json-schema.org/draft-07/schema"
+
+	// Known JSON Schema properties
+
+	JSONSchemaIDProperty           = "$id"
+	JSONSchemaAdditionalIDProperty = "id"
+	JSONSchemaSchemaProperty       = "$schema"
+	JSONSchemaNameProperty         = "name"
+	JSONSchemaDescriptionProperty  = "description"
```

Comment:
I'm surprised that these are needed.

Reply:
these are in anticipation of https://github.com/w3c/vc-json-schema/issues/146